importer/DeHeDe.py

```diff
@@ -0,0 +1,90 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class DeHeDe(Monument):
+
+    def set_location(self):
+        if self.has_non_empty_attribute("ortsteil"):
+            location_q = utils.q_from_wikipedia("de", self.ortsteil)
+            if location_q:
+                self.add_statement("location", location_q)
+
+    def set_adm_location(self):
+        city_q = utils.q_from_wikipedia("de", self.stadt)
+        self.add_statement("located_adm", city_q)
+
+    def update_descriptions(self):
+        placename = None
+        germany = {
+            "de": "Deutschland",
+            "en": "Germany",
+            "sv": "Tyskland"
+        }
+        templates = {
+            "en": "Kulturdenkmal in {}",
+            "de": "Kulturdenkmal in {}",
+            "sv": "kulturminnesmärke i {}"
+        }
+        for lg in templates:
+            country = germany[lg]
+            if self.has_non_empty_attribute("stadt"):
+                placename = utils.get_rid_of_brackets(self.stadt)
+                placename = "{}, {}".format(placename, country)
+            else:
+                placename = country
+            desc = templates[lg].format(placename)
+            self.add_description(lg, desc)
+
+    def update_labels(self):
+        if self.has_non_empty_attribute("bezeichnung"):
+            german = utils.remove_markup(self.bezeichnung)
+        elif self.has_non_empty_attribute("adresse"):
+            german = utils.remove_markup(self.adresse)
+        self.add_label("de", german)
```

Comment:
is it guaranteed that one or the other always exists?

Reply:
Well, there's nothing technical to guarantee that items with both these fields empty don't make their way into the database at some point, but at least in my snapshot no, there's none with both fields missing. As for the numbers, there's 8500 (out of 17k) with an empty Bezeichnung, as well as 220 with an empty Adresse.

Reply:
Works for me. Thanks for checking.